### DISCUSSION TITLE
Önkiszolgáló egységen UPS

### DISCUSSION THREAD

**Discussion by laurelkft** [nav-gov-hu/eRECEIPT#6] [Ötletek]
Jelenleg kötelező az önkiszolgáló kasszán az UPS.
Valóságban viszont ha elmegy az áram, akkor már sokkal nagyobb a probléma, senki sem azzal lesz elfoglalva, hogy befejezze a tranzakciót...
Javasoljuk a kötelező UPS kivételét a szabályzatból, és a piacra, vagy a  megoldás szállítójára bízni, hogy ez valóban szükséges-e, vagy sem.
Nem a fejlesztés nagy kihívás, hanem feleslegesnek tartjuk, hogy ha az épület eleve rendelkezik szünetmentes hálózattal, akkor még egy extra UPS legyen beépítve az önkiszolgálóba. Bizonyos kültéri esetekben ez nem is életképes. (pl. benzinkutaknál önkiszolgáló kútfejbe integrálni egy olyan eszközt, ami azért meghibásodás esetén rossz csillagállás esetén akár ki is gyulladhat, ezzel extra kockázatot okozva. Lényegében nem szabad ilyen eszközt integrálni a kútfejekbe, így a jelenlegi szabályzás is az oka annak, hogy bizonyos helyeken a két eszközt (kútfej és OPT) elszeparálva kell telepíteni egymástól)

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
